tsl/test/sql/CMakeLists.txt

```
@@ -60,6 +60,7 @@ set(TEST_FILES_DEBUG
   transparent_decompression_queries.sql
   tsl_tables.sql
   license.sql
+  backup.sql
```

Since the test is about distributed case, I suggest to follow other distributed tests and prefix with `dist_`.
```suggestion
  dist_backup.sql
```
Can it be also moved into the place with other distributed tests, i.e., follow lexicographical order.